Studio-Lovelies/GG-JointJustice-Unity
# Deduct default volume from GameAudioMixer + Music-Slider works + Dan sprites

## Summary
Noticed and fixed a few things:
- the `Music` slider in the settings wasn't working
- without save data, the game set the mixer to full volume; instead it now takes defaults we set in the editor
- adjusted the levels again
- added a few of Dan's new sprites

## Testing instructions
1. Start the game outside the editor and play for a bit
2. Notice how your eardrums remain intact
3. Open the editor
4. Start the game in-editor and change audio settings
5. Restart the game and notice your settings still being applied
6. Select `Edit -> Clear All PlayerPrefs`
7. Change levels in the `GameAudioMixer`
8. Start the game
9. Notice how the default audio levels of the AudioMixer are applied as no SaveData is present (before this PR, they would've reset to -0dB)

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
